sdk/search/azure-search-documents/src/main/java/com/azure/search/documents/SearchIndexingBufferedSenderOptions.java

```diff
@@ -70,17 +92,17 @@ public boolean getAutoFlush() {
      * The buffered sender will reset the duration when documents are sent for indexing, either by reaching {@link
      * #setInitialBatchActionCount(int)} or by a manual trigger.
      * <p>
-     * If {@code flushWindow} is negative or zero and {@link #setAutoFlush(boolean)} is enabled the buffered sender will
-     * only flush when {@link #setInitialBatchActionCount(int)} is met.
+     * If {@code autoFlushInterval} is negative or zero and {@link #setAutoFlush(boolean)} is enabled the buffered
+     * sender will only flush when {@link #setInitialBatchActionCount(int)} is met.
      *
-     * @param autoFlushWindow Duration between document batches being sent for indexing.
+     * @param autoFlushInterval Duration between document batches being sent for indexing.
      * @return The updated SearchIndexingBufferedSenderOptions object.
-     * @throws NullPointerException If {@code autoFlushWindow} is null.
+     * @throws NullPointerException If {@code autoFlushInterval} is null.
      */
-    public SearchIndexingBufferedSenderOptions<T> setAutoFlushWindow(Duration autoFlushWindow) {
-        Objects.requireNonNull(autoFlushWindow, "'autoFlushWindow' cannot be null.");
+    public SearchIndexingBufferedSenderOptions<T> setAutoFlushInterval(Duration autoFlushInterval) {
```

Comment:
Is the API name change not a breaking change here?
Or Moving from beta to beta changeset?

Reply:
Beta update